Common/Securities/Forex/Forex.cs

```diff
@@ -47,7 +47,8 @@ public Forex(SecurityExchangeHours exchangeHours, Cash quoteCurrency, Subscripti
                 new ImmediateSettlementModel(),
                 Securities.VolatilityModel.Null,
                 new SecurityMarginModel(50m),
-                new ForexDataFilter()
+                new ForexDataFilter(),
+                new AdjustedPriceVariationModel()
```

Comment:
Yes movement towards more accuracy is positive even if it changes previous backtests.